x-pack/plugins/cases/README.md

```diff
@@ -140,16 +140,6 @@ Arguments:
 UI component:
 ![Recent Cases Component][recent-cases-img]
 
-## Case Action Type
```

Comment:
Could you please remove everything from here to the bottom of the file 🙂? All the sections below belong to the  cases connector.

Reply:
Oh sorry I was confused about which was for the case connector 😆